cmd/api/book/book_test.go

```diff
@@ -12,6 +15,22 @@ import (
 	gomock "go.uber.org/mock/gomock"
 )
 
+var ctx context.Context = context.Background()
+
+func TestMain(m *testing.M) {
+	var err error
+
+	reqTimeoutStr := os.Getenv("HTTP_REQUEST_TIMEOUT") //This ENV must be written with a unit suffix, like seconds
+	if reqTimeoutStr != "" {
+		book.RequestTimeout, err = time.ParseDuration(reqTimeoutStr)
+		if err != nil {
+			log.Fatalln("getting request timeout from env: %w", err)
+		}
+	}
```

Comment:
For testing proposes you don't need to read from the environment, you can set it directly so you can control it.